tests/show_techsupport/test_techsupport.py

```diff
@@ -0,0 +1,253 @@
+import pytest
+import os
+import pprint
+from loganalyzer import LogAnalyzer, LogAnalyzerError
+import time
+from random import randint
+from log_messages import *
+import logging
+logger = logging.getLogger(__name__)
+
+DEFAULT_LOOP_RANGE = 10
+DEFAULT_LOOP_DELAY = 10
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+ACL_TMP_DIR = os.path.basename('acl_tmp')
+
+FILES_DIR = os.path.join(BASE_DIR, 'files')
+TEMPLATE_DIR = os.path.join(BASE_DIR, 'templates')
+
+ACL_RULES_FULL_TEMPLATE = 'acltb_test_rules.j2'
```

Comment:
Maybe we should use same naming convention
1. ACL_RULES_FULL_TEMPLATE = 'acltb_test_rules.j2'
This one called template, while others as file. what is the difference?
2. ACL_RULE_PERSISTENT_J2 = 'acl_rule_persistent.json.j2
This onc called J2 where the file is the same but this has json, the first not. Is it by purpose

Suggest you go over the name of the files you created and the defines you used and see if you can align with same concept.